src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -15,7 +15,9 @@ module Dialogs
     class PartitionSize < CWM::Dialog
       # @param disk_name [String]
       # @param ptemplate [#region]
```

Comment:
this is actually not true. IT uses much more methods then just `#region` as it is passed to SizeWidget which require almost all methods from ptemplate.